pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -78,6 +78,9 @@ def __init__(self, adapter, **kwargs):
         "FREQ:STOP %s",
         "Stop frequency in Hz.",
     )
+    nominal_level = Instrument.control(
+        "POW:RLEV?", "POW:RLEV %s", "Defines the nominal level of the R&S FSW"
+    )
```

Comment:
Not really a frequency setting. I think this is only defined for the phase noise channel type so maybe should get its own section.

Reply:
Resolved via [697127a](https://github.com/bleykauf/pymeasure/pull/4/commits/697127a14c501fae622095b3e2ec04cabfc76fdd)